index.js

```diff
@@ -1,23 +1,39 @@
 'use strict'
 
-module.exports = function objectifyArray (array, keyName) {
+module.exports = function objectifyArray (array, keys, options) {
```

Comment:
What about making `keyNames` an option, with a default value of `['id']`?

Reply:
I had originally coded it to look like `objectify(x, { by: 'id' })`, but it felt a bit verbose as I was revising the examples in the readme. That usually doesn't bother me, but that plus library precedence was enough to push back to what you see in the PR.